internal/oci/annotations.go

```diff
@@ -0,0 +1,147 @@
+	// OCI spec.
+	//
+	// Note: This annotation is in MB. OCI is in Bytes. When using this override
+	// the caller MUST use MB or sizing will be wrong.
+	AnnotationMemorySizeInMB         = "io.microsoft.virtualmachine.computetopology.memory.sizeinmb"
+	AnnotationMemoryLowMMIOGapInMB   = "io.microsoft.virtualmachine.computetopology.memory.lowmmiogapinmb"
+	AnnotationMemoryHighMMIOBaseInMB = "io.microsoft.virtualmachine.computetopology.memory.highmmiobaseinmb"
+	AnnotationMemoryHighMMIOGapInMB  = "io.microsoft.virtualmachine.computetopology.memory.highmmiogapinmb"
+	// annotationProcessorCount overrides the hypervisor isolated vCPU count set
+	// via the OCI spec.
+	//
+	// Note: Unlike Windows process isolated container QoS Count/Limt/Weight on
+	// the UVM are not mutually exclusive and can be set together.
+	AnnotationProcessorCount = "io.microsoft.virtualmachine.computetopology.processor.count"
+	// annotationProcessorLimit overrides the hypervisor isolated vCPU limit set
+	// via the OCI spec.
+	//
+	// Limit allows values 1 - 100,000 where 100,000 means 100% CPU. (And is the
+	// default if omitted)
+	//
+	// Note: Unlike Windows process isolated container QoS Count/Limt/Weight on
+	// the UVM are not mutually exclusive and can be set together.
+	AnnotationProcessorLimit = "io.microsoft.virtualmachine.computetopology.processor.limit"
+	// annotationProcessorWeight overrides the hypervisor isolated vCPU weight set
+	// via the OCI spec.
+	//
+	// Weight allows values 0 - 10,000. (100 is the default if omitted)
+	//
+	// Note: Unlike Windows process isolated container QoS Count/Limt/Weight on
+	// the UVM are not mutually exclusive and can be set together.
+	AnnotationProcessorWeight             = "io.microsoft.virtualmachine.computetopology.processor.weight"
+	AnnotationVPMemCount                  = "io.microsoft.virtualmachine.devices.virtualpmem.maximumcount"
+	AnnotationVPMemSize                   = "io.microsoft.virtualmachine.devices.virtualpmem.maximumsizebytes"
+	AnnotationPreferredRootFSType         = "io.microsoft.virtualmachine.lcow.preferredrootfstype"
+	AnnotationBootFilesRootPath           = "io.microsoft.virtualmachine.lcow.bootfilesrootpath"
+	AnnotationKernelDirectBoot            = "io.microsoft.virtualmachine.lcow.kerneldirectboot"
+	AnnotationVPCIEnabled                 = "io.microsoft.virtualmachine.lcow.vpcienabled"
+	AnnotationVPMemNoMultiMapping         = "io.microsoft.virtualmachine.lcow.vpmem.nomultimapping"
+	AnnotationStorageQoSBandwidthMaximum  = "io.microsoft.virtualmachine.storageqos.bandwidthmaximum"
+	AnnotationStorageQoSIopsMaximum       = "io.microsoft.virtualmachine.storageqos.iopsmaximum"
+	AnnotationFullyPhysicallyBacked       = "io.microsoft.virtualmachine.fullyphysicallybacked"
+	AnnotationDisableCompartmentNamespace = "io.microsoft.virtualmachine.disablecompartmentnamespace"
+	AnnotationVSMBNoDirectMap             = "io.microsoft.virtualmachine.wcow.virtualSMB.nodirectmap"
```

Comment:
Now that we're going to export all of these, we'll want to have comments for these now.

Reply:
done